src/components/Word/index.tsx

```diff
@@ -113,6 +115,7 @@ export type WordProps = {
   onFinish: Function
   isStart: boolean
   wordVisible: boolean
+  switchTips: any
```

Comment:
Why `any`?

Reply:
sorry, I'll replace it with correct type later